modules/build/src/main/scala/scala/build/options/SNNumeralVersion.scala

```diff
@@ -0,0 +1,38 @@
+package scala.build.options
+
+case class SNNumeralVersion(major: Int, minor: Int, patch: Int) {
+
+  override def equals(other: Any): Boolean =
```

Comment:
Since SNNumberalVersion is a case class it contains the same implementation of equals by default